pages/AccountPage.ts

```diff
@@ -0,0 +1,36 @@
+import { expect, Page, Locator } from "@playwright/test";
+
+export class AccountPage {
+  readonly page: Page;
+  readonly pageTitle: Locator;
+  readonly navMenu: Locator;
+
+  constructor(page: Page) {
+    this.page = page;
+    this.pageTitle = page.getByTestId("page-title");
+    this.navMenu = page.getByTestId("nav-menu");
+  }
+
+  async waitForLoad(): Promise<void> {
+  await expect(this.pageTitle).toBeVisible();
+}
+  async getPageTitleText() {
+    return await this.pageTitle.textContent();
+  }
+
+  async getNavMenuText() {
+    return await this.navMenu.textContent();
```

Comment:
await is not necessary here 
```suggestion
    return this.navMenu.textContent();
```